.travis.yml

```diff
@@ -7,7 +7,10 @@ script:
 before_deploy:
 - wget -O setup https://github.com/triplea-game/triplea/blob/master/.travis/setup?raw=true
 - chmod +x setup && ./setup
-- export TAGGED_VERSION="$BUILD_VERSION.$TRAVIS_BUILD_NUMBER"
```

Comment:
Note that 'BUILD_VERSION' is no longer used, this is a redundant place where we had the game engine version. Otherwise below we read from "game_engine.properties", parse out the version, set that to "ENGINE_VERSION", then we append the build number, and write it back out to the game_engine.properties file.